win32/service.py

```diff
@@ -176,6 +177,13 @@ def SvcDoRun(self):
         for proc in self.procs.values():
             proc.start()
 
+        # check to see if apm is enabled.
+        if self.config["apm_enabled"]:
```

Comment:
Nit: it'd be safer to set `self.config = {}` in the `__init__` and change this to `if self.config.get('apm_enabled'):`